executor/poolmgr/gp.go

```diff
@@ -715,3 +723,27 @@ func (gp *GenericPool) destroy() error {
 
 	return nil
 }
+
+func (gp *GenericPool) getFetcherResources() (v1.ResourceRequirements, error) {
```

Comment:
Extract this to a common/util file + it doesn't seem to be needed to attach this function the GenericPool.